src/main/java/org/jfrog/hudson/util/SpecUtils.java

```diff
@@ -2,33 +2,54 @@
 
 import hudson.EnvVars;
 import hudson.FilePath;
+import hudson.Plugin;
 import hudson.Util;
+import hudson.model.AbstractBuild;
+import hudson.model.BuildListener;
+import jenkins.model.Jenkins;
 import org.apache.commons.lang.StringUtils;
 import org.jfrog.hudson.SpecConfiguration;
 
 import java.io.IOException;
 import java.io.PrintStream;
 
+
 /**
  * Created by diman on 16/10/2016.
  */
 public class SpecUtils {
 
-    public static String getSpecStringFromSpecConf(SpecConfiguration specConfiguration, EnvVars env, FilePath workspace, PrintStream logger)
+    public static String getSpecStringFromSpecConf(SpecConfiguration specConfiguration, AbstractBuild<?, ?> build, final BuildListener listener)
             throws IOException, InterruptedException {
-
+        EnvVars env =build.getEnvironment(listener);
         if (StringUtils.isNotBlank(specConfiguration.getFilePath())) {
-            String filePath = specConfiguration.getFilePath().trim();
-            filePath = Util.replaceMacro(filePath, env);
-            String spec = buildDownloadSpecPath(filePath, workspace, logger).readToString();
-            return Util.replaceMacro(spec.trim(), env);
+            String filePath = expand(build, listener, specConfiguration.getFilePath().trim());
+            FilePath workspace= build.getExecutor().getCurrentWorkspace();
+            PrintStream logger =listener.getLogger();
```

Comment:
Let's keep the spacing around the = sign according to the convention:
a = 1 (OK)
a =1 (not OK).
There are additional places in the code which should also be corrected.